crates/bins/src/bin/datadog-static-analyzer-git-hook.rs

```diff
@@ -159,6 +159,12 @@ fn main() -> Result<()> {
     opts.optflag("t", "include-testing-rules", "include testing rules");
     opts.optflag("", "secrets", "enable secrets detection (BETA)");
     opts.optflag("", "static-analysis", "enable static-analysis");
+    opts.optopt(
+        "",
+        "timeout",
```

Comment:
an option should give a meaning of what is does. with the unit

`rule-exec-timeout-ms` is better for example: it include the what (e.g. `rule execution`) and includes the unit (`millisecond`).

Reply:
improved the name, but I disagree with the unit being in the flag name itself.